### ISSUE TITLE
new release including search stall bugfix 

### ISSUE THREAD

**Issue by dpsutton** [technomancy/leiningen#2114]
The latest release was 2.6.1 from feb 9. It looks like there was a bug relating to a missing content length header which caused the clojars index building to fail for searching. This was quickly patched and much appreciated.

However, the standard way to install lein from the readme is a bash script which downloads the most recent released version of lein. However, this points to 2.6.1 which predates the search stall fix. I was wondering if you could create a new release so that the standard build chain would then have this patch and I could continue learning clojure and lein. Today is my first day looking at it and I'm working through the lein tutorial but cannot search at the moment.

Problem: search is broken but fixed; fix is not included in latest release version
     `LEIN_URL="https://github.com/technomancy/leiningen/releases/download/$LEIN_VERSION/leiningen-$LEIN_VERSION-standalone.zip"`

Fix: create a new release, then lein script can be modified `export LEIN_VERSION="2.6.1"` so that we can get this necessary patch.

**Comment by hyPiRion** [comment]
I think the current changes does not warrant a new release just yet, but I agree that we should release more frequently. I've set the due date for [the 2.6.2 milestone](https://github.com/technomancy/leiningen/milestones/2.6.2) to 31th of March, and I'll do a release before then regardless of how much we've completed.

If anyone wants to help out, then verifying that the windows .bat pull requests looks and works alright would be helpful. Looking into and fixing unassigned issues will be really helpful as well.
